public/app/features/alerting/unified/components/expressions/util.ts

```diff
@@ -37,10 +40,20 @@ const formatLabels = (labels: Labels): string => {
     .join(', ');
 };
 
+const formatFolderName = (namespace: CombinedRuleNamespace): string => {
```

Comment:
nit: prefer to rename this to `decodeGrafanaNamespace` (or similar) with some context around what we're doing here, why we're doing it (nested folder, etc)